src/sections/LocationDetails.astro

```diff
@@ -22,13 +25,15 @@ const location = {
 			rating: 4.3,
 			googleReviews: 1417,
 			image: doubletree,
+			link: 'https://www.hilton.com/en/hotels/ontcldt-doubletree-claremont/'
 		},
 		{
 			name: 'Sheraton Fairplex Suites & Conference Center',
 			address: '601 W McKinley Ave, Pomona, CA 91768',
 			rating: 4.4,
 			googleReviews: 1930,
 			image: sheraton,
+			link: 'https://www.marriott.com/en-us/hotels/ontsi-sheraton-fairplex-suites-and-conference-center/overview/'
 		},
 	],
```

Comment:
are these the correct hotels from Scott? I think they're the same as when the conference was going to be located at San Dimas, but now that there's a new location idk if these still make sense

Reply:
Nope. Let's confirm these are the correct hotels.